tests/aggregation/tests.py

```diff
@@ -839,6 +839,37 @@ def test_values_aggregation(self):
             max_books_per_rating,
             {'books_per_rating__max': 3})
 
+    def test_ticket26390(self):
+        """
+        Check that the random call does not break GROUP BY cluse when
+        used as ordering.
+        """
+        qs = Author.objects.annotate(contact_count=Count('book'))
+
+        self.assertQuerysetEqual(qs.order_by('contact_count'), [
+            ('Adrian Holovaty', 1),
```

Comment:
Yes, I missed that. I’ve added `ordered=False` here since ordering is not the point here anyway.